config/cnv_report_template.html

```diff
@@ -319,6 +319,12 @@ <h2>Results table</h2>
                     <p>
                         BAF values outside the range 0&ndash;1 might indicate that the tumor cell content is underestimated.
                     </p>
+                    <p>
+                        Note that variants labelled as COPY_NORMAL can be present in the filtered table.
+                        This indicates that there is a variant covering the same gene that has been called by a different caller.
```

Comment:
```suggestion
                        This indicates that there is a variant covering the same gene that has been called by a different caller or the variant has a VAF value < 0.3 or > 0.7 .
```

Reply:
I was a bit reluctant to hardcode something that can be changed in the config, but at the same time there is no straightfoward way of extracting this information. Let's go with this for now.